.jenkins/jenkins-functions.groovy

```diff
@@ -98,7 +98,6 @@ def postAction(String coresDir, String artifactDir) {
     if (status == null) {
         currentBuild.result = 'SUCCESS'
     }
-    tearDownOpenshift()
```

Comment:
We don't need this for PRs etc?

Reply:
nope, since openstack server is deleted immediately after test run